src/Service/VendorService/TheMovieDatabase/TheMovieDatabaseVendorService.php

```diff
@@ -72,70 +143,233 @@ protected function extractData(object $jsonContent): array
         return $data;
     }
 
+    /**
+     * Guess the work type (movie or tv-show) based on the datawell result.
+     *
+     * @TODO Currently has high number of tv-show that gets marked as movies.
+     *
+     * @param array $subjects
+     *
+     * @return WorkType
+     */
+    private function getWorkType(array $subjects): WorkType
+    {
+        $searchDkcdPlus = ['dkdcplus:DK5-Text', 'dkdcplus:DBCS', 'dkdcplus:genre'];
+
+        $types = [];
+
+        foreach ($subjects as $subject) {
+            if (property_exists($subject, '@type')) {
+                if (in_array($subject->{'@type'}->{'$'}, $searchDkcdPlus)) {
+                    $types[] = mb_strtolower($subject->{'$'});
+                }
+
+                if ('dcterms:LCSH' === $subject->{'@type'}->{'$'}) {
+                    if (str_ends_with($subject->{'$'}, ' films')) {
+                        $types[] = 'spillefilm';
+                    }
+                }
+            }
+        }
+
+        // Order is important. A TV Show will also have the
+        // subject "Spillefilm" but a movie will not have the
+        // subject "tv-serier".
+        // @TODO match by title also and look for "Disc, Episode, etc"?
+        if (in_array('tv-serier', $types)) {
+            return WorkType::TV_SHOW;
+        }
+
+        if (in_array('spillefilm', $types)) {
+            return WorkType::MOVIE;
```

Comment:
Why not set `WorkType::MOVIE` to `spillefilm` and the same for `tv-serier`?

Reply:
Not understood. Do you want `WorkType`to be a backed enum with `case MOVIE = spillefilm;`? Then `WorkType` will be specific to the datawell. Wanted a general type that works for both The Movie database and the datawell